include/zephyr/drivers/misc/devmux/devmux.h

```diff
@@ -13,6 +13,7 @@
 #define INCLUDE_ZEPHYR_DRIVERS_MISC_DEVMUX_H_
 
 #include <stdint.h>
+#include <sys/types.h>
```

Comment:
Ugh - sorry - I should have fixed this a while ago.

I think it would be best to just return an int rather than `ssize_t`  from the function below.

Reply:
This is simply fixing an existing problem. Please open a separate issue if you would like to drive this change.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/79898